UN-OCHA/ocha_ai
# [RW-1016] Reduce size of AI chat popup icon

Refs: RW-1016

This reduces the size of the popup icon to 64px instead of 80px.

## Tests

1. Make sure you are using this branch of the module
2. Clear the browser cache
3. Visit a report page on your local RW site
4. Confirm the round button with the icon is 64x64px
